views/add_event.hbs

```diff
@@ -1,79 +1,143 @@
-      <option value="Carer">Carer</option>
-      <option value="Disabled parking">Disabled Parking</option>
-      <option value="Disabled toilets">Disabled Toilets</option>
-      <option value="Good lighting">Good Lighting</option>
-      <option value="Place for guide dog">Place for Guide Dog</option>
-      <option value="Sign language">Sign Language</option>
-      <option value="SMS messaging">SMS Messaging</option>
-      <option value="Wheelchair access">Wheelchair Access</option>
-    </select>
+    <div class="checkbox-wrap">
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Audio recordings">Audio Recordings</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Big fonts">Big Fonts</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Braille">Braille</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Carer">Carer</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Disabled parking">Disabled Parking</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Disabled toilets">Disabled Toilets</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Good lighting">Good Lighting</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Place for guide dog">Guide Dog</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Sign language">Sign Language</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="SMS messaging">SMS Messaging</input></label>
+      <label><input
+      type="checkbox"
+      name="accessibilityOptions"
+      value="Wheelchair access">Wheelchair Access</input></label>
+    </div>
 
     <label class="sel1">Categories </label>
-    <select multiple class="options form-control" name="categories" required>
-      <option disabled selected>Select option..</option>
-      <option value="competition">Competition</option>
-      <option value="conference">Conference</option>
-      <option value="culture">Culture</option>
-      <option value="dining">Dining</option>
-      <option value="educational">Educational</option>
-      <option value="launch">Launch</option>
-      <option value="miscellaneous">Miscellaneous</option>
-      <option value="music">Music</option>
-      <option value="party">Party</option>
-      <option value="sport">Sport</option>
-      <option value="wedding">Wedding</option>
-    </select>
-
-    <label>Place </label>
-    <div id="addPlaceIcon" class="glyphicon glyphicon-plus"></div>
+    <div class="checkbox-wrap">
+      <label><input
```

Comment:
Sorry to be a pain, but any chance you could lay it out like this? Looks weird atm.

```html
<label>
   <input ... />
</labe>
```